friendsofsymfony1/swiftmailer/CVE-2024-28859.yaml

```diff
@@ -0,0 +1,14 @@
+title:     Deserialization Gadget chain in Swift Mailer
+link:      https://github.com/FriendsOfSymfony1/symfony1/security/advisories/GHSA-wjv8-pxr6-5f4r
+cve:       CVE-2024-28859
+branches:
+    4.x:
+        time:     null
+        versions: ['>=4.0.0', '<=4.3.1']
+    5.x:
+        time:     null
```

Comment:
this time should be the time of the backport done in https://github.com/FriendsOfSymfony1/swiftmailer/pull/18 (the time of the merge to be exact)